src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
+    import Base.maximum, Base.minimum
 
-    export update!, state, nobs, rand, ci
+    export update!, state, nobs, rand, ci, sample
```

Comment:
We need to derive `sample` from `StatsBase`. Exporting it will clobber what exists in `StatsBase`.